### ISSUE TITLE
IE11 error 

### ISSUE THREAD

**Issue by danibs** [dbfannin/ngx-logger#60]
Hi,
I'm using the last version (but also with version 2.0.0) there is a problem with IE11 on class NGXLoggerUtils:

```
static getCallerDetails(): {lineNumber: string, fileName: string} {
    const err = (new Error(''));

    // this should produce the line which NGX Logger was called
const callerLine = err.stack.split('\n')[4].split('/');
```

this is the error:
![immagine](https://user-images.githubusercontent.com/30932554/38920497-ade33c3a-42f3-11e8-8c95-bb0a2d57c0df.png)

**Comment by dbfannin** [comment]
@danibs, this should be fixed in ngx-logger@2.2.1.

**Comment by wcdalton** [comment]
Any idea why I would see this error again?

IE 11, Version 11.557.17763.0
ngx-logger:3.4.2

Polyfills:
import "core-js/es6/symbol";
import "core-js/es6/object";
import "core-js/es6/function";
import "core-js/es6/parse-int";
import "core-js/es6/parse-float";
import "core-js/es6/number";
import "core-js/es6/math";
import "core-js/es6/string";
import "core-js/es6/date";
import "core-js/es6/array";
import "core-js/es6/regexp";
import "core-js/es6/map";
import "core-js/es6/weak-map";
import "core-js/es6/set";
import "core-js/es7/array";

![NGXLogger-issue](https://user-images.githubusercontent.com/13040566/60837671-f7b3f880-a185-11e9-92d7-4b7f474615a7.PNG)

**Comment by wcdalton** [comment]
Solved in 3.4.3 I see.
